doc/source/examples/pdp_regression_bike.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<p>This paragraph probably needs rewriting. Would be cool to motivate the need for 2-way plots here (can refer to Molnar's exposition).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>On second thought this is ok, I would just change the start of the sentence to "We continue with some examples...".</p>